README.md

```diff
@@ -38,6 +38,7 @@ It collect key metrics about:
 | rds_allocated_storage_bytes | `aws_account_id`, `aws_region`, `dbidentifier` | Allocated storage |
 | rds_api_call_total | `api`, `aws_account_id`, `aws_region` | Number of call to AWS API |
 | rds_backup_retention_period_seconds | `aws_account_id`, `aws_region`, `dbidentifier` | Automatic DB snapshots retention period |
+| rds_ca_certificate_valid_until | `aws_account_id`, `aws_region`, `dbidentifier` | Timestamp of the expiration of the RDS Certificate Authority |
```

Comment:
Following [Prometheus best practices](https://prometheus.io/docs/practices/naming/), we should use a base unit like `seconds`

Similar to [Prometheus Blackbox exporter](https://github.com/prometheus/blackbox_exporter/blob/a7d02da341bd5995eb7f6a1f1be7cff68d99397c/prober/prober.go#L39-L42), I would suggest to use `rds_certificate_expiry_timestamp_seconds`

Reply:
Valid point, I've edited my commit to use `rds_certificate_expiry_timestamp_seconds`.